storage/store.go

```diff
@@ -102,27 +121,90 @@ func initializeDB(db *sql.DB, cfg Configuration, embeddedScripts embed.FS) error
 			}
 			sqlStatements = append(sqlStatements, string(sqlStatement))
 		}
-	} else {
-		f, err := embeddedScripts.ReadFile("sqlscripts/0001-jaeger-index.sql")
+	case cfg.Replication:
```

Comment:
This seems unnecessarily duplicated. We ca have a single code to read the scripts and perhaps just change the paths?

Reply:
> This seems unnecessarily duplicated.
I think so too, but I don't know how to make it better, 'cause I pass different params every time.